templates/zerver/help/include/sidebar_index.md

```diff
@@ -141,6 +141,7 @@
 * [Add a custom linkifier](/help/add-a-custom-linkifier)
 * [Message retention policy](/help/message-retention-policy)
 * [SAML authentication](/help/saml-authentication)
+* [Restrict GIFs rating from GIPHY](/help/giphy-rating)
```

Comment:
I think it'd be better for this to just be a section at the bottom of "Animated GIFs from GIPHY".